### DISCUSSION TITLE
Is it possible to use Azure OpenAI API key for Assistant?

### DISCUSSION THREAD

**Discussion by Tom-Boscher** [zed-industries/zed#10131] [Feature Requests]
I have a lot of Azure credits I don't use so I would like to use them for Zed Assistant but I don't know how to setup the Azure OpenAI API key to use it in Assistant.

**Comment by tgds** [comment]
I'm puzzled as well. It seems there was support for it introduced in #8646, but then quietly removed in #7367. I'd love to have that brought back.

**Comment by baldwindavid** [comment]
I'm also unsure of quite what is going on, but if you [search `assistant2` in pull requests](https://github.com/zed-industries/zed/pulls?q=is%3Apr+assistant2+is%3Aclosed) _things are happening_. Perhaps this will open up other sources like Azure, Copilot Chat, etc. 🤷

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
